gcc/ChangeLog:

2019-04-24  Iain Buclaw  <ibuclaw@gdcproject.org>

        * config.gcc (*-*-dragonfly*): Add dragonfly-d.o
        * config/dragonfly-d.c: New file.
        * config/t-dragonfly: New file.

libphobos/ChangeLog:

2019-04-24  Iain Buclaw  <ibuclaw@gdcproject.org>

        * configure.tgt: Add *-*-dragonfly* as a supported target.

---diff --git gcc/config.gcc gcc/config.gcc
index 09fb9ecd2cd..14f0a1abbc8 100644
--- gcc/config.gcc
+++ gcc/config.gcc
@@ -724,6 +724,9 @@ case ${target} in
   extra_options="$extra_options rpath.opt dragonfly.opt"
   default_use_cxa_atexit=yes
   use_gcc_stdint=wrap
+  d_target_objs="${d_target_objs} dragonfly-d.o"
+  tmake_file="${tmake_file} t-dragonfly"
+  target_has_targetdm=yes
   ;;
 *-*-freebsd*)
   # This is the generic ELF configuration of FreeBSD.  Later
